src/tar-utils.ts

```diff
@@ -20,11 +22,17 @@ async function getTarCompressionMethod(): Promise<CompressionMethod> {
       silent: true,
     })
     .then((out) => out.stdout.trim())
-    .then((out) => [out, semver.clean(out)]);
+    .then((out) => {
+      const extractedVersion = /(v\d+(?:\.\d+){0,})/.exec(out);
```

Comment:
let's not capture the `v` prefix in the regexp if we compare with `1.3.2` (without the `v` aswell)?

Reply:
Yup totally make sense. Edited the history.